### PR TITLE
Footer-Section

### PR DESCRIPTION
**Pull request by MoHassn**
[The link that shows my changes(My Site)](https://mohassn.charity.m3ntorship.com/)
[Ticket link on Clickup](https://app.clickup.com/t/4pkwxn)

### CODE REVIEW

File: dist/assets/css/footer.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.footer {
+  color: #f1feff;
```
Comment (Nilegfx) [inline dist/assets/css/footer.css:2]: consider using existing global tailwind colors, or add this color to the global if it does not exist and then use it.

Comment (MoHassn) [reply dist/assets/css/footer.css:2]: Done

**Comment by abdalla-elmahdy** [comment]
Beware, before you pull from master again, backup your CSS file because it will get deleted right away.

**Comment by MoHassn** [comment]
> please add the background image like in the psd

Could you please provide me with the background image?
or just tell me where I can find it.

**Comment by elhaw** [comment]
> > please add the background image like in the psd
> 
> Could you please provide me with the background image?
> or just tell me where I can find it.
**I guess you mean the footer background-image**
If not check this URL [clickup ticket for assets](https://app.clickup.com/t/689nmg)
footer-hands-bg

**Pull request merged**